### ISSUE TITLE
failing tests

### ISSUE THREAD

**Issue by maxharris9** [burnpiro/puppeteer-screenshot-tester#12]
**Describe the bug**
Tests failing on master

**Please complete the following information:**
 - OS: macOS 10.13.5
 - Node v10.11.0

**To Reproduce**
1. clone this repo locally
2. `npm i`
3. `npm t`

**Expected behavior**
Passing tests

**Stack Trace**
```
Maxs-MacBook-Pro:puppeteer-screenshot-tester max$ npm t

> puppeteer-screenshot-tester@1.0.4 test /Users/max/nodesource/puppeteer-screenshot-tester
> jest

 FAIL  examples/google.test.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
          at Array.forEach (<anonymous>)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.577s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

**Additional context**

**Comment by burnpiro** [comment]
I've manage to fix that by upgrading to puppeteer 1.11.0 but for some reason there is an issue when running puppeteer in headless mode on google.com which fails on:
```
    const browser = await puppeteer.launch()
   [...]
    await page.type('input[title="Search"]', 'Hello', { delay: 100 })
```
but works on non-headless mode
```
    const browser = await puppeteer.launch({headless: false})
   [...]
    await page.type('input[title="Search"]', 'Hello', { delay: 100 })
```

Message is `No node found for selector: input[title="Search"]`. I'll create issue on puppeteer repo and see what they have to say. For now i'll change it to run test with {headless: false}.

**Comment by burnpiro** [comment]
I've got answer from puppeteer team, they've told me that is not really puppeteer issue and it's related with google's website. Main reason why headless mode works differently is because google tries to recognize user agent and location in non-headless browsers and servers different site :(

I could change test to use sth else than google but because it's only example we could deal with it.
